### PR TITLE
[FIX] sales_team migration error if crm is not installed

### LINKED ISSUE #729: [8.0] sales_team error (relation "crm_case_section" does not exist)
Impacted versions: 8.0 (target upgrade version)

Steps to reproduce:
* install a fresh V7.0 database, -i point_of_sale,purchase. (i guess that less module is sufficient, but not tested)
* upgrade to V8.0

Current behavior:
* During the migration, sales_team (new 8.0 module is installed by [dependencies](https://github.com/OCA/OpenUpgrade/blob/8.0/addons/sales_team/__openerp__.py#L32)).
* sales_team migration is executed.
* an error is raised :

        error in migration script OpenUpgrade/addons/sales_team/migrations/8.0.1.0/pre-migration.py: 
        relation "crm_case_section" does not exist
        LINE 1: UPDATE crm_case_section SET code = '_DM' WHERE code = 'DM'

Expected behavior:
No error.

Analyse & Solution:
Part of the crm migration should be executed early. That's why part of the code is in sales_team. But migration should include test to know if crm is installed.

### PR DESCRIPTION
**Pull request by legalsylvain**
Description of the issue/feature this PR addresses:
FIX #729

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/sales_team/migrations/8.0.1.0/post-migration.py [hunk 1]
```diff
@@ -20,12 +20,19 @@
...
 ##############################################################################
 
+import logging
 from openerp import pooler, SUPERUSER_ID
 from openerp.openupgrade import openupgrade, openupgrade_80
 
+logger = logging.getLogger('OpenUpgrade.sales_team')
+
 
 @openupgrade.migrate(no_version=True)
```
Comment (pedrobaeza) [inline addons/sales_team/migrations/8.0.1.0/post-migration.py:30]: Better to remove `no_version` and then the scripts are not executed on module install (which is your case). As you were the original author of the module, maybe you had a reason for putting that.

Comment (legalsylvain) [reply addons/sales_team/migrations/8.0.1.0/post-migration.py:30]: Hi @pedrobaeza. 
Thanks a lot for your review.

- sales_team doesn't exist in V7.0.
- this module will be installed on V8.0, by dependencies.
- this migration script should be executed if crm is installed on V7.0.
- this migration script should not be executed if crm is not installed on V7.0.

so no_version should stay, but crm test is necessary.

What do you think ?

Comment (pedrobaeza) [reply addons/sales_team/migrations/8.0.1.0/post-migration.py:30]: OK, understood, but then move this to crm migration scripts...

Comment (legalsylvain) [reply addons/sales_team/migrations/8.0.1.0/post-migration.py:30]: No. it's too late in crm. crm.case.section moved from crm module (V7.0) to sales_team module (V8.0).

https://github.com/OCA/OpenUpgrade/blob/8.0/addons/sales_team/sales_team.py#L28

kind regards.

**Comment by pedrobaeza** [comment]
Ok then

**Pull request merged**